tests/disk/test_disk_exhaustion.py

```diff
@@ -170,22 +170,22 @@ def test_disk_exhaustion(duthost, ptfadapter, tbinfo, creds):
     # Create a shell script to do the operations like fallocate and remove file,
     # because when space is full, duthost.command() is not work
 
-    # Get available space in /tmp mounted partition, the output of "df /tmp" was like below:
+    # Get available space in /host mounted partition, the output of "df /host" was like below:
     #   Filesystem     1K-blocks    Used Available Use% Mounted on
     #   root-overlay    14874056 6429908   8427764  44% /
-    df_rst = duthost.shell("df /tmp")["stdout_lines"][1].split()
+    df_rst = duthost.shell("df /host")["stdout_lines"][1].split()
     available_kb = int(df_rst[3])
     allocate_kb = int(available_kb * 0.95)
     used_before_test = int(df_rst[4].rstrip('%'))
     logger.info(f"Preparing to allocate ~{allocate_kb} KB (95% of available) to simulate disk exhaustion")
 
     # Setup test.sh and execute
     duthost.shell_cmds(cmds=[
-        f"echo 'fallocate -l {allocate_kb}K /tmp/huge_dummy_file' > /tmp/test.sh",
-        "echo 'sleep 60' >> /tmp/test.sh",
-        "echo 'sudo rm -f /tmp/huge_dummy_file' >> /tmp/test.sh",
-        "chmod u+x /tmp/test.sh",
-        "nohup /tmp/test.sh >/dev/null 2>&1 &"
+        f"echo 'fallocate -l {allocate_kb}K /host/huge_dummy_file' > /host/test.sh",
+        "echo 'sleep 60' >> /host/test.sh",
```

Comment:
This test should include the SSH connect and IP packet forward test, do we know the typical duration for these tests ? is the 60s enough or is it too limiting, we should avoid wasting time but also ensure the test could be completed. thanks

Reply:
Actually, the test is quite fast, 60s is enough to finish.